packages/@aws-cdk/core/lib/from-cfn.ts

```diff
@@ -0,0 +1,29 @@
+import { CfnResource } from './cfn-resource';
+
+/**
+ * An interface that represents callbacks into a CloudFormation template.
+ * Used by the fromCloudFormation methods in the generated L1 classes.
+ *
+ * @experimental
+ */
+export interface ICfnFinder {
+  /**
+   * Returns the resource with the given logical ID in the template.
+   * If a resource with that logical ID was not found in the template,
+   * returns undefined.
+   */
+  findResource(logicalId: string): CfnResource | undefined;
+}
+
+/**
+ * The interface used as the last argument to the fromCloudFormation
+ * static method of the generated L1 classes.
+ *
+ * @experimental
+ */
+export interface FromCloudFormationProps {
```

Comment:
For an operation, it's generally `Options`. It's only `Props` for the constructor properties of a construct.

Reply:
Of course, you're right. Will fix.